ic-dev-base/Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV HOME=/workspace
 WORKDIR $WORK_DIR
 # Dfinity SDK
 ARG motoko_version=0.6.29
-ARG dfx_version=0.9.3
+ARG dfx_version=0.11.1-beta.3
```

Comment:
Are you sure we want to use a beta version?

Reply:
Yes, sure. It was also discussed with @blutooth 
BTW current pipelines will use previously built images with DFX 0.9.3